apis/r/vignettes/soma-experiment-queries.Rmd

```diff
@@ -104,16 +172,29 @@ query$n_obs
 As before, we can load the `obs` data frame into memory but now it only includes the filtered observations.
 
 ```{r}
-query$obs(column_names = c("obs_id", "nCount_RNA"))$to_data_frame()
+as.data.frame(query$obs(column_names = c("obs_id", "nCount_RNA"))$concat())
```

Comment:
Minor thing, but we've been using `$to_data_frame()` elsewhere in the vignettes. I don't have a strong preference for either approach but would like to be consistent to minimize cognitive burden on readers.

Reply:
I'll refractor to `$to_data_frame()`